pkg/block/fetcher.go

```diff
@@ -401,14 +401,25 @@ func NewLabelShardedMetaFilter(relabelConfig []*relabel.Config) *LabelShardedMet
 	return &LabelShardedMetaFilter{relabelConfig: relabelConfig}
 }
 
+const blockIDLabel = "__block_id"
+
 // Filter filters out blocks that filters blocks that have no labels after relabelling.
```

Comment:
This comment is quite confusing to me. Do the filtered blocks filter out other blocks with no labels? or is there an extra `filters out blocks` here?

Reply:
Yes I think there is a double `filters out blocks`.